### ISSUE TITLE
FUTURE (Stage 2) - Rollovers for menu buttons. 

### ISSUE THREAD

**Issue by DrewFellows** [toastcreative/kanebridge#12]
Add rollover states to menu buttons.

**Comment by DrewFellows** [comment]
Are you able to create a simple rollover state for every button with CSS or do you need me to create artwork for each button. 

![rollover example](https://cloud.githubusercontent.com/assets/15023716/10535044/bd12ceae-742a-11e5-9bd0-934e83500b1f.jpg)

**Comment by mark-toast** [comment]
Yes actually I really need you to create a PSD for each and every button state ;p
